src/App.jsx

```diff
@@ -14,70 +15,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [activeGood, setActiveGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const Good = ({ good }) => {
+    const isActiveGood = good === activeGood;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+    return (
+      <tr
+        data-cy="Good"
+        className={isActiveGood ? 'has-background-success-light' : ''}
+      >
+        <td>
+          {isActiveGood ? (
             <button
               data-cy="RemoveButton"
               type="button"
               className="button is-info"
+              onClick={() => setActiveGood(null)}
             >
               -
             </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
+          ) : (
             <button
               data-cy="AddButton"
               type="button"
               className="button"
+              onClick={() => setActiveGood(good)}
             >
               +
             </button>
-          </td>
+          )}
+        </td>
+
+        <td data-cy="GoodTitle" className="is-vcentered">
+          {good}
+        </td>
+      </tr>
+    );
+  };
+
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {activeGood ? `${activeGood} is selected` : 'No goods selected'}
+
+        {activeGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setActiveGood(null)}
```

Comment:
you write `() => setActiveGood(null)` twice - create a separate function for it and reuse